spec/models/order_item_spec.rb

```diff
@@ -229,5 +230,80 @@
         expect(item.total_item_price).to eq(@product.price * quantity)
       end
     end
+
+    context "#adjust_if_product_stock_changed!" do
+      it "reduces the quantity ordered if product has less stock" do
+        product = Product.create(name: 'asdafun94ymc34', price: 1, seller_id: 1, stock: 1)
```

Comment:
I _really_ prefer to use `@seller.id` rather than `1` (etc with any id), but it's fine if you want to leave it like this.